src/main/resources/hudson/plugins/emailext/ExtendedEmailPublisher/config.groovy

```diff
@@ -72,9 +73,9 @@ f.advanced(title: _("Advanced Settings")) {
     f.checkbox(name: "project_save_output", checked: instance?.saveOutput)
   }
 
-  def configuredTriggers = instance != null ? instance.configuredTriggers : [hudson.plugins.emailext.plugins.trigger.FailureTrigger.createDefault()]
+  def configuredTriggers = instance != null ? instance.configuredTriggers : [hudson.plugins.emailext.plugins.trigger.FailureTrigger.createDefault(),hudson.plugins.emailext.plugins.trigger.UnstableTrigger.createDefault(),hudson.plugins.emailext.plugins.trigger.StatusChangedTrigger.createDefault()]
```

Comment:
Having only failures as a trigger initially is the basic setup similar to Mailer, people expect emails for failures by default, but not emails for other things. I had additional triggers enabled by default at one point and people complained about the number of emails being sent by the default configuration.